Sources/PkgConfig/PkgConfig.swift

```diff
@@ -10,12 +10,21 @@
 
 import Utility
 import func POSIX.getenv
+import func POSIX.popen
 
 enum PkgConfigError: ErrorProtocol {
     case CouldNotFindConfigFile
     case ParsingError(String)
 }
 
+/// Get search paths from pkg-config itself.
+/// This is needed because on Linux machines, the search paths can be different
+/// from the standard locations that we are currently searching
```

Comment:
this line needs full stop too ( [llvm](http://llvm.org/docs/CodingStandards.html) conventions.. )